index.bs

```diff
@@ -2881,15 +2889,14 @@ Given an [=attribution rate-limit record=] |newRecord|:
     [=set/union|unioned=] with «|newRecord|'s [=attribution rate-limit record/reporting origin=]»
 1. If |distinctReportingOrigins|'s [=list/size=] is greater than |max|, return <strong>blocked</strong>.
 
-    NOTE: [=rate-limit scope/source=] scopes have an auxiliary [=max source reporting origins per source reporting site=] rate limit that also must be enforced.
+To <dfn>check if an [=attribution source=] should be blocked by reporting-origin per site limit</dfn> given an [=attribution source=] |source|:
```

Comment:
Why is this algorithm defined under the `Triggering algorithms` section if it is used during source-processing?

Reply:
This is pre-existing, but moved to general algorithm section.